### PR TITLE
update macaroons/README.md

### PR DESCRIPTION
**Pull request by tlindi**
Provide sample of command to create macaroon with similar rights as admin.macaroon to be used with 3rd party applications.

## Change Description
Description of change / link to associated issue.

## Steps to Test
Steps for reviewers to follow to test the change.

### Testing
- [ ] Your PR passes all CI checks.
- [ ] Tests covering the positive and negative (error paths) are included.
- [ ] Bug fixes contain tests triggering the bug to prevent regressions.

### Code Style and Documentation
- [X ] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [ ] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ] Any new lncli commands have appropriate tags in the comments for the rpc in the proto file.
- [ ] [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

**Comment by tlindi** [comment]
@coderabbitai review

File: macaroons/README.md [hunk 1]
```diff
@@ -114,6 +114,22 @@ The macaroon created by this call would only be allowed to call the `GetInfo` an
...
 
+Might you need a macaroon file with rights similar to admin.macaroon for custom 
+use case you can create such like in this example:
+
+```shell
+$  lncli bakemacaroon --save_to lnbits.macaroon \
+   address:read address:write \
+   info:read info:write \
+   invoices:read invoices:write \
+   macaroon:generate macaroon:read macaroon:write \
```
Comment (guggero) [inline macaroons/README.md:125]: Might make sense to add a sentence that when `macaroon:generate` is granted, then that macaroon can be used to create **new** macaroons with more permissions than the original macaroon. So this permission should be used with caution.

**Review by guggero — APPROVED** [review]
Thanks for the doc improvement!

File: macaroons/README.md (hunk 1 above)
Comment (tlindi) [reply macaroons/README.md:127]: Added.

**Comment by tlindi** [comment]
https://github.com/coderabbitai review

**Comment by tlindi** [comment]
@coderabbitai review

**Comment by tlindi** [comment]
Improved grammar.

**Comment by tlindi** [comment]
Shortened lines under 80 cols.

**Review by yyforyongyu — APPROVED** [review]

**Pull request merged**